test/include/armhf_emulation.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+overlayfs_dir=""
+is_dev_setup_mounted=false
+
+setup_armhf_emulation()
+{
+    set -e
+    image_file="${1}"
+    target_root_dir="${2}"
+
+    echo "Setup armhf emulation."
+
+    # devtmps needs to be re-mounted when running in docker
+    if ! grep -qE "/dev.*devtmpfs" "/proc/mounts"; then
+        mount -t "devtmpfs" "none" "/dev"
+        is_dev_setup_mounted=true
+    fi
+
+    # overlayfs is required because chroot expects the qemu binary in /bin
+    # and modified partition tables are created in this test
```

Comment:
not sure if we had this coment here before, but it's not entirey true, we don't test anything, nor modify any partition tables here :p, also the kernel expects the arm emulation binary there, even in a chrooted environment.